aten/src/ATen/native/nested/NestedTensorUtils.h

```diff
@@ -50,6 +50,22 @@ inline at::Tensor wrap_buffer(
       std::move(offsets));
 }
 
+inline at::Tensor wrap_buffer(
+    at::Tensor buffer,
+    at::Tensor nested_size_tensor,
+    at::Tensor nested_stride_tensor,
+    const std::vector<int64_t>& offsets) {
+  std::vector<int64_t> offsets_copy;
+  offsets_copy.resize(offsets.size());
+  for (size_t i = 0; i < offsets.size(); i++) {
```

Comment:
is this more efficient than calling the std::vector<int64_t> copy constructor?

Reply:
I don't know. I haven't gotten to the profile + improve stage yet.